pages/college_predictor.js

```diff
@@ -36,10 +45,18 @@ const CollegePredictor = () => {
                 );
 
                 // Sort the filteredData in ascending order of college rank
+                // for same college (with same rank), sort with opening rank
                 filteredData.sort((a, b) => {
                     const rankA = a["College Rank"];
```

Comment:
Use better variable names than `rankA` and `rankB` . Also, shouldn't we be sorting by closing rank?

Reply:
closing rank filtering happens in previous lines to obtain `filteredData`

within filtered colleges -- we display colleges with better college rank.
now, if college rank is same (ie, rows with same college but different branch) -- we sort using opening rank [that is, of the branch. typically CSE opening rank better than EE opening rank]